draft-ietf-jsonpath-base.md

```diff
@@ -1386,6 +1386,8 @@ normal-element-index      = "0" / (DIGIT1 *DIGIT) ; non-negative decimal integer
 | `$.a` | `$['a']` | Object value |
 | `$[1]` | `$[1]`  | Array index |
 | `$.a.b[1:2]` | `$['a']['b'][1]` | Nested structure |
+| `$["\u000B"]`| `$['\u000b']` | Unicode escape |
+| `$["\u0061"]`| `$['a']` | Unicode character |
```

Comment:
I would have preferred a special symbol (e.g. `±`) or unusual character (e.g. `å`), but I couldn't find one which rendered correctly in text and HTML.

Reply:
See my fix (this is probably as good as it gets with RFCXMLv3)